ros/cpp/src/ptg_datahub/src/ptg_datahub.cpp

```diff
@@ -113,6 +126,11 @@ class MinimalPublisher : public rclcpp::Node
       unsigned int frames_received = 0;
 
       publisher_ = this->create_publisher<sensor_msgs::msg::Image>(PORT_TOPIC_MAP[port], 10);
+
+      if (port == PV_TCP_PORT)
+      {
+        pose_publisher_ = this->create_publisher<angel_msgs::msg::HeadsetPoseData>("HeadsetPoseData", 10);
```

Comment:
Red flag: Defining a constant in the body of the code. Instead create a constant up at the top with everything else. Eventually when all this stuff becomes variables, then we don't have to hunt for things like this.